*** menu.pl.orig	Wed Jan 25 13:59:41 1995
--- menu.pl	Wed Jan 25 14:58:50 1995
***************
*** 134,143 ****
  $kill_buffer    = "";
  
  # PERL5 ONLY -- Uncomment these statements if you DON'T have "getcap()"
! #use Term::Cap qw(Tgetent %TC);		# Get Tgetent package
! #$Term::Cap::ospeed = 9600;		# Suppress "nospeed" warning
! #Term::Cap::Tgetent();			# Define entry
! #sub main'getcap { $Term::Cap::TC{shift}; }	# Define local subroutine
  
  #**********
  #  MENU_CURSES_APPLICATION
--- 134,146 ----
  $kill_buffer    = "";
  
  # PERL5 ONLY -- Uncomment these statements if you DON'T have "getcap()"
! #               OR if the demo doesn't appear to work (there's a bug in
! #               some getcap()'s).
! #
! #require Term::Cap;			# Get Tgetent package
! #$Term::Cap::ospeed = 9600;		# Suppress "nospeed" warning
! #Term::Cap::Tgetent();			# Define entry
! #*main'getcap = sub { $Term::Cap::TC{shift} };	# Define local subroutine
  
  #**********
  #  MENU_CURSES_APPLICATION
