*** lib_newterm.c.orig	Sun Jan  1 22:13:43 1995
--- lib_newterm.c	Mon Jan  2 01:53:26 1995
***************
*** 40,46 ****
  SCREEN * newterm(char *term, FILE *ofp, FILE *ifp)
  {
  sigaction_t act;
! int	errret;
  int	stolen, topstolen;
  
  #ifdef TRACE
--- 40,46 ----
  SCREEN * newterm(char *term, FILE *ofp, FILE *ifp)
  {
  sigaction_t act;
! int	errret, i;
  int	stolen, topstolen;
  
  #ifdef TRACE
***************
*** 73,78 ****
--- 73,81 ----
  	SP->_fifohead	= -1;
  	SP->_fifotail 	= 0;
  	SP->_fifopeek	= 0;
+ 
+ 	for (i = 0; i < FIFO_SIZE; i++)
+ 		SP->_fifo[i] = 0;
  
  	if (enter_ca_mode)
  	    	putp(enter_ca_mode);
